src/common/helpers/notify.js

```diff
@@ -32,6 +35,14 @@ async function sendEmail(templateId, emailAddress, personalisation = {}) {
     await notifyClient.sendEmail?.(templateId, emailAddress, {
       personalisation
     })
+    audit({
+      event: {
+        category: AUDIT_EVENT_CATEGORIES.EMAIL,
+        action: AUDIT_EVENT_ACTIONS.EMAIL_SENT
+      },
+      template_id: templateId,
+      email_address: emailAddress
```

Comment:
Nitpick: Can I just check that we're explicitly not putting the these values in `data` or `context` object? If so, what were the reasons for that decision?

Reply:
It makes sense to put our custom payload in such an object. I've changed it now. 
I think we should we standardise it for this project and document it.